speculation-rules.bs

```diff
@@ -51,6 +51,7 @@ spec: html; urlPrefix: https://html.spec.whatwg.org/multipage/
       text: get an element's target; url: get-an-element's-target
     urlPrefix: links.html
       text: following a hyperlink; url: following-hyperlinks-2
+      text: url; url: concept-hyperlink-url
```

Comment:
It doesn't really make a difference, but it might be nicer to do `for: HTMLHyperlinkElementUtils` and then change the call sites to `[=HTMLHyperlinkElementUtils/url=]`.

Reply:
Thanks, done!